packages/react/src/__tests__/ReactChildren-test.js

```diff
@@ -903,16 +925,14 @@ describe('ReactChildren', () => {
 
     const container = document.createElement('div');
     const root = ReactDOMClient.createRoot(container);
-    await expect(async () => {
-      await act(() => {
-        root.render(
-          <ComponentRenderingMappedChildren>
-            <div />
-            <div />
-          </ComponentRenderingMappedChildren>,
-        );
-      });
-    }).toErrorDev([]);
+    await act(() => {
+      root.render(
+        <ComponentRenderingMappedChildren>
+          <div />
+          <div />
+        </ComponentRenderingMappedChildren>,
+      );
+    });
```

Comment:
no error here anymore?

Reply:
Yeah it's confusing but the `toErrorDev()` helper is a console.error, and since the array was empty it means there were no console errors. The assert helpers throw if a test ends with un-asserted logs, so no assertion is needed.